cmd/dm-master/main_test.go

```diff
@@ -32,6 +34,12 @@ func TestRunMain(t *testing.T) {
 		}
 	}
 
+	defer func() {
+		if err := recover(); err != nil {
```

Comment:
if we `panic` in a separate goroutine, this `recover` in the main goroutine cannot recover it.
Can this achieve our goals?

Reply:
yep, this still need to solve, I will dig into it later.